tools/tools.go

```diff
@@ -18,8 +18,18 @@
 package tools
 
 import (
-	_ "github.com/dnephin/govet"
-	_ "k8s.io/code-generator"
-	_ "k8s.io/gengo"
-	_ "k8s.io/klog"
+	// Because in 1.12.x there is no go files udner k8s.io/code-generator, so
+	// we need to import these commands explicitly.
+	// TODO import k8s.io/code-generator only in k8s.io 1.15+
```

Comment:
- `github.com/dnephin/govet` is not used anymore
- no need to vendor `k8s.io/klog` and `k8s.io/gengo`
- in 1.12.x, we should vendor k8s.cio/code-generator/cmd/xxx directly